### PR TITLE
Main

### PR DESCRIPTION
**Pull request by jase-d**
Add features to main page and adjust sizing/margins
[Check it out here](https://recordit.co/z4raBIBvC7)

A lot of refactoring and moving around, feel free to skim

@bland0 
@javsflores

### CODE REVIEW

**Review by javsflores — APPROVED** [review]
Looking good, Jase. I like all the cleaning you did importing a lot of your css to different files.

**Comment by jonathandavidlewis** [comment]
Heads up, you can make the image visible here with markdown by downloading the GIF from recordit.co and then dragging it into your PR description or any comment in github:
image_example

File: client/dist/css/Animations.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#main-photo {
+  filter: opacity(0);
+  animation-name: carouselStart;
+  animation-duration: .8s;
+  animation-delay: .5s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-play-state: initial;
+}
+
```
Comment (jonathandavidlewis) [inline client/dist/css/Animations.css:19]: nit: extra line

File: client/dist/css/General.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+
+  #ms-1 {
+    border-top-right-radius: 6%;
+  }
+
+  #ms-3 {
+    border-bottom-right-radius: 6%;
+  }
```
Comment (jonathandavidlewis) [inline client/dist/css/General.css:56]: Add blank line at end of file

Comment (jonathandavidlewis) [reply client/dist/css/General.css:56]: every text file

**Comment by jase-d** [comment]
That’s awesome I’m going to try that for next time.

File: client/src/components/modal/photo_feed.jsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 /////////////////////////////////////////////
```
Comment (jonathandavidlewis) [inline client/src/components/modal/photo_feed.jsx:1]: See file naming convention

File: client/src/components/mosaic-main/header.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+//////////////////////////////////
+import React from 'react';
+//////////////////////////////////
+import {MainStyles} from '../../styles/Mosaic.js';
+//////////////////////////////////
+const MockHeader = () => (
```
Comment (jonathandavidlewis) [inline client/src/components/mosaic-main/header.jsx:6]: File Naming MockHeader.jsx
https://github.com/airbnb/javascript/tree/master/react#naming

**Pull request merged**